libcore/Directory.vala

```diff
@@ -531,14 +579,13 @@ public class Files.Directory : Object {
 
     private static void toggle_ref_notify (void* data, Object object, bool is_last) {
         if (is_last) {
-            unowned Directory dir = (Directory) object;
-            debug ("Directory is last toggle_ref_notify %s", dir.file.uri);
-
+            var dir = (Directory) object;
```

Comment:
```suggestion
            unowned var dir = (Directory) object;
```

Reply:
Can we keep the owned var and remove the dir.ref () instead? Is it equivalent?